### PR TITLE
Add sporco recipe

### PR DESCRIPTION
**Pull request by bwohlberg**
Initial recipe for sporco package. The recipe passes a local build test and appears to be ready to merge.

### CODE REVIEW

File: recipes/sporco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # Skip windows build since pyfftw dependency currently not available
```
Comment (jentjr) [inline recipes/sporco/meta.yaml]: Please remove comments.
```suggestion

```

File: recipes/sporco/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  # noarch: python
```
Comment (jentjr) [inline recipes/sporco/meta.yaml]:
```suggestion

```

File: recipes/sporco/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - pytest-runner
+  run:
+    - future
+    - python
+    - numpy >=1.12
+    - scipy >=0.13.3
+    - imageio
+    - matplotlib >=1.3.1
+    - pyfftw >=0.10.4
+
```
Comment (jentjr) [inline recipes/sporco/meta.yaml]: Delete extra space.
```suggestion
```

File: recipes/sporco/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    - pip
+    - numpy >=1.12
+    - scipy >=0.13.3
+    - imageio
+    - pytest
+    - pytest-runner
+  run:
+    - future
+    - python
+    - numpy >=1.12
```
Comment (jentjr) [inline recipes/sporco/meta.yaml]: Please pin compatible numpy.
```suggestion
    - {{ pin_compatible('numpy') }}
```

File: recipes/sporco/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+requirements:
+  host:
+    - future
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.12
+    - scipy >=0.13.3
+    - imageio
+    - pytest
```
Comment (jentjr) [inline recipes/sporco/meta.yaml]: Are pytest and pytest-runner needed to build the package, or are they only needed for testing?

File: recipes/sporco/meta.yaml (hunk 4 above)
Comment (bwohlberg) [reply recipes/sporco/meta.yaml]: My understanding from the docs is that `pin_compatible('numpy')` is intended for packages that link against numpy, but this package just imports it as a module. Is `pin_compatible('numpy')` really appropriate here?

File: recipes/sporco/meta.yaml (hunk 5 above)
Comment (bwohlberg) [reply recipes/sporco/meta.yaml]: In principle they're only needed for the tests, but the most recent release on PyPi has pytest-runner listed as install requirements, so it's needed here too, for now. I've removed the pytest entry.

File: recipes/sporco/meta.yaml (hunk 4 above)
Comment (jentjr) [reply recipes/sporco/meta.yaml]: No, it is not appropriate then. Not pinning numpy will actually reduce the number of builds. Sorry for the confusion.

**Comment by jentjr** [comment]
@conda-forge/staged-recipes I believe this is ready to go. Thanks!

**Comment by bwohlberg** [comment]
Thanks for the review @jentjr.

**Pull request merged**